src/order/category/LinearOrder.lean

```diff
@@ -34,8 +39,8 @@ instance : inhabited LinearOrder := ⟨of punit⟩
 
 instance (α : LinearOrder) : linear_order α := α.str
 
-instance has_forget_to_PartialOrder : has_forget₂ LinearOrder PartialOrder :=
-bundled_hom.forget₂ _ _
+instance has_forget_to_Lattice : has_forget₂ LinearOrder Lattice :=
+{ forget₂ := { obj := λ X, Lattice.of X, map := λ X Y, order_hom.to_lattice_hom } }
```

Comment:
@Vierkantor, I can't get `order_hom_class.to_lattice_hom` to work here because it doesn't want to unfold the types 😢 
The precise error is
```
failed to synthesize type class instance for
X Y : LinearOrder,
f : X ⟶ Y
⊢ order_hom_class (X ⟶ Y) ↥(Lattice.of ↥X) ↥(Lattice.of ↥Y)
```